src/app/table-filters/table-filters.component.ts

```diff
@@ -41,4 +43,12 @@ export class TableFiltersComponent implements OnInit {
             }, {});
         this.router.navigate([this.router.url.split(';')[0], queryParams]);
     }
+
+    getFilter(property: string): string {
+        const parameter = this.router.url.split(';').find(param => {
+            return param.includes(property);
+        });
+
+        return parameter ? decodeURI(parameter.replace(`${property}=`, '')) : '';
```

Comment:
`return decodeUri(this.router.url.split(';').find(param.includes(property)) || '')`